frontera/contrib/backends/memory/__init__.py

```diff
@@ -128,11 +135,11 @@ def _get(self, obj):
 
     def update_cache(self, objs):
         objs = objs if type(objs) in [list, tuple] else [objs]
-        map(self._put, objs)
+        list(map(self._put, objs))
```

Comment:
the map was used here, because in Py2.7 it's faster than iterate with for cycle. According to my tests the 
```python
[f(x) for x in data] # is faster than

list(map(f, data))
```

in python 3.5
So I'm suggesting to rewrite it that way.